docs/examples/basic_crawler.ts

```diff
@@ -8,16 +7,16 @@ const dataset = await Dataset.open();
 // users to implement the crawling logic themselves.
 const crawler = new BasicCrawler({
     // This function will be called for each URL to crawl.
-    async requestHandler({ request }) {
+    async requestHandler({ request, sendRequest }) {
         const { url } = request;
         console.log(`Processing ${url}...`);
 
-        // Fetch the page HTML via Apify utils gotScraping
-        const { body } = await gotScraping({ url });
+        // Fetch the page HTML via the crawlee sendRequest utility method
+        const { body } = await sendRequest(request);
```

Comment:
worth mentioning that `sendRequest()` will do the same here, as it takes the contextual `request` object as default value